tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.PowerShell.Cmdlets.Storage.dll.json

```diff
@@ -242503,6 +242503,20 @@
                 "Parameters": [],
                 "ReturnType": "System.Void"
               },
+              {
```

Comment:
The 2 json file change should be removed from this PR:

1. They will take difficulty when merge master to preview
2. They are for breaking change tracking, not need add